golden_hour/location.py

```diff
@@ -1,6 +1,8 @@
 from astral import Astral, Location
 import schema
 
+# Lowest reasonable elevation is Death Valley, about 100m below sea level.
```

Comment:
Haha, now I'm wondering about highest allowable elevation 😄 Everest is 8,848 meters.

Reply:
or you could be in spaaaace